hack/make_local.py

```diff
@@ -18,7 +18,7 @@ def gen_local_deployments(outdir):
         csv = yaml.safe_load(csv_file)
 
     with open(f'{outdir}/local.yaml', 'w') as out:
-        csv = [x for x in csv['spec']['install']['spec']['deployments'] if x['name'] != 'hco-operator']
+        csv = [x for x in csv['spec']['install']['spec']['deployments'] if x['name'] != 'hco-operator' and x['name'] != 'hco-webhook' ]
```

Comment:
I'm not sure about this line.

originally, we wanted to debug the operator. Now we have two applications to debug, but usually we'll want to debug only one of them.

I guess most of the cases, the developer will want to debug the operator, but will still need the webhook in place. And if the developer wants to debug the webhook, it still required a running operator.

So I think we need to have two flags (e.g. using envarinmet variables) to independently run the operatrot and/or the webhook. e.g. `DEBUG_OPERATOR` and `DEBUG_WEBHOOK`.

Reply:
added DEBUG_OPERATOR and DEBUG_WEBHOOK variables into the script for this purpose